apax/nodes/model.py

```diff
@@ -57,11 +57,11 @@ def _post_load_(self) -> None:
         self._handle_parameter_file()
 
     def _handle_parameter_file(self):
-        with self.state.use_tmp_path():
-            self._parameter = yaml.safe_load(pathlib.Path(self.config).read_text())
+        self._parameter = yaml.safe_load(self.state.fs.read_text(self.config))
 
+        with self.state.use_tmp_path():
             custom_parameters = {
-                "directory": self.model_directory.resolve().as_posix(),
+                "directory": self.model_directory.as_posix(),
```

Comment:
How come the resolve is not needed any more?

Reply:
Maybe it was never required?

Reply:
it was definitely required. I think it might have been for the case of manually restarting the MD from within the node directory